quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: quarkus.jdt.ls Test Plugin
+Bundle-SymbolicName: com.redhat.microprofile.jdt.quarkus.test
+Bundle-Version: 0.8.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.junit,
+ org.apache.commons.io,
+ org.apache.commons.lang3,
+ org.eclipse.jdt.ls.core,
+ com.google.guava,
+ org.eclipse.lsp4j,
+ org.eclipse.lsp4j.jsonrpc,
+ org.eclipse.core.runtime,
+ org.eclipse.jdt.core,
+ org.eclipse.jdt.core.manipulation,
+ org.eclipse.core.resources,
+ org.eclipse.m2e.core,
+ org.eclipse.m2e.maven.runtime,
+ org.eclipse.m2e.lifecyclemapping.defaults,
+ org.eclipse.buildship.core,
+ org.eclipse.ltk.core.refactoring,
+ org.eclipse.text,
+ com.redhat.microprofile.jdt.core,
+ com.redhat.microprofile.jdt.test,
+ com.redhat.microprofile.jdt.quarkus
+Import-Package: com.google.gson
+Export-Package: com.redhat.microprofile.jdt.core,
+ com.redhat.microprofile.jdt.core.contextpropagation,
+ com.redhat.microprofile.jdt.core.faulttolerance,
+ com.redhat.microprofile.jdt.core.health,
+ com.redhat.microprofile.jdt.core.jaxrs,
+ com.redhat.microprofile.jdt.core.lra,
+ com.redhat.microprofile.jdt.core.metrics,
+ com.redhat.microprofile.jdt.core.openapi,
+ com.redhat.microprofile.jdt.core.opentracing,
+ com.redhat.microprofile.jdt.core.restclient
```

Comment:
I don't think we should export those packages. They might be exported by the microprofile.jdt.test plugin, if needed by @jeffmaury